<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>413</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Million Live Quintet Player</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QCheckBox" name="idolactive3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>278</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>278</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive0">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>278</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>278</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>278</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="positionSlider">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>480</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="idolSlider">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>445</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="BGMSlider">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>445</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel0">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>510</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>510</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>510</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QComboBox" name="songsel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>153</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="soloButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>11</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Solo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="unitButton">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>11</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Unit</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>50</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Song:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>419</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>BGM Volume</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>419</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Idol Volume</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage0">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="randomizeButton">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>510</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>148</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive11">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>148</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>148</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>148</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive12">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>408</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>408</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive8">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>408</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="idolsel10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="idolactive10">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>408</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="idolimage8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>73</width>
      <height>73</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="unitButton13">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>11</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>13</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
